bazel/README.md

```diff
@@ -0,0 +1,2 @@
+Please notice that any bazel targets and definitions in this repository are currently experimental and for internal use
```

Comment:
Can we move this to the top of the workspace?
I think it would be much more discoverable there.

Reply:
I put it both in `CONTRIBUTING.md` and as a comment to `WORKSPACE.bazel` :+1: